server/network.go

```diff
@@ -94,12 +99,20 @@ func newLocalNetwork(opts localNetworkOptions) (*localNetwork, error) {
 		return nil, err
 	}
 
+	pluginDir := ""
+	if opts.execPath != "" {
```

Comment:
Does it matter which of these has precedence?

Reply:
yes. opts.pluginDir has precedence over opts.execPath. it enables to give some exec path and some other non standard plugin dir.